api/controllers/productsController.js

```diff
@@ -86,3 +87,15 @@ exports.getByPriceRange = async (request, response) => {
 		response.status(error.code).json(error)
 	}
 }
+
+exports.generateProductsCatalog = async (request, response) => {
+	try {
+		const generatedPdf = await productsCatalog(request.body)
+
+		setTimeout(() => {
```

Comment:
why do you need timeout here?

Reply:
cuz the file is being created too long so without it you'd have to click generation button twice